observability/metrics/src/main/scala/com/daml/metrics/ExecutorServiceMetrics.scala

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import java.util.concurrent.{ExecutorService, ForkJoinPool, ThreadPoolExecutor}
+
+import com.daml.metrics.api.MetricHandle.Factory
+import com.daml.metrics.api.{MetricName, MetricsContext}
+import org.slf4j.LoggerFactory
+
+class ExecutorServiceMetrics(factory: Factory) {
+
+  private val logger = LoggerFactory.getLogger(getClass)
+  private val prefix = MetricName("daml", "executor")
+  private val PoolMetricsPrefix: MetricName = prefix :+ "pool"
+  private val TasksMetricPrefix: MetricName = prefix :+ "tasks"
```

Comment:
For consistency, we should either have all of these suffixed like `MetricsPrefix` or `MetricPrefix`.
```suggestion
  private val TasksMetricsPrefix: MetricName = prefix :+ "tasks"
```